reconcile/test/test_gitlab_labeler.py

```diff
@@ -0,0 +1,142 @@
+from unittest import TestCase
+from unittest.mock import patch, create_autospec
+
+from typing import Any, Optional
+
+from reconcile.queries import APPS_QUERY
+import reconcile.gitlab_labeler as gl
+from reconcile.utils import gql
+from .fixtures import Fixtures
+
+
+class TestData:
+    '''Class to add data to tests in setUp. It will be used by mocks'''
+    def __init__(self):
+        self._apps = []
+
+    @property
+    def apps(self) -> list[dict]:
+        return self._apps
+
+    @apps.setter  # type: ignore[no-redef, attr-defined]
+    def apps(self, apps: list[dict]) -> None:
+        if not isinstance(apps, list):
+            raise TypeError(f'Expecting list, have {type(apps)}')
+        self._apps = apps
+
+
+class TestOnboardingGuesser(TestCase):
+
+    def setUp(self) -> None:
+        self.test_data = TestData()
+
+        self.fxt = Fixtures('apps')
+        self.app1 = {'path': '/services/parent/normalapp/app.yml',
```

Comment:
why don't you get these from the above yaml fixtures?

Reply:
It's possible that I implemented something wrong, but with the fixtures I'm not capable of expanding the parent app of an app as the real graphql is doing, that's why I ended up using "hardcoded" apps

Reply:
Then you will need to return exactly what qontract-server is returning for the query you're mocking, `APPS_QUERY`. In order to do that I get the exact json doc that q-s returns.